ush/parsing_namelists_FV3.sh

```diff
@@ -31,10 +34,32 @@ EOF
 cat $DIAG_TABLE >> diag_table
 fi
 
+[ ! -z "${AERO_DIAG_TABLE}" ] && cat ${AERO_DIAG_TABLE} >> diag_table
```

Comment:
I'm trying to eliminate these short-circuit constructs in favor of proper if-blocks.

Reply:
Agreed.

Reply:
Done in d94ef47ab971fd5097c8bccaf6c4e521d057dc54